src/index.jsx

```diff
@@ -197,17 +206,46 @@ const init = async ({
     replaceTitleOnMobile,
     isPublic,
     onLogOut,
-    userActionRequired: getUserActionRequired()
+    userActionRequired: getUserActionRequired(),
+    reduxStore
   })
 }
 
 const updateAccessToken = accessToken => {
   stack.updateAccessToken(accessToken)
 }
 
+// Handle exceptions for API before init
+const apiReferences = {}
+APILocations.forEach(location => {
```

Comment:
We use references here, it allows to get the method available when the application will 'import/require' it but it will look for the real `exposedAPI` only on demand when the application will call the method.